Objects/weakrefobject.c

```diff
@@ -29,6 +30,7 @@ init_weakref(PyWeakReference *self, PyObject *ob, PyObject *callback)
     self->wr_next = NULL;
     Py_XINCREF(callback);
     self->wr_callback = callback;
+    self->vectorcall = weakref_vectorcall;
```

Comment:
To fix compiler warnings:

```suggestion
    self->vectorcall = (vectorcallfunc)weakref_vectorcall;
```